snmp/assets/dashboards/troubleshooting_dashboard.json

```diff
@@ -0,0 +1,2533 @@
+{
+    "author_name": "Datadog",
+    "description": null,
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "Host",
+            "prefix": "host"
```

Comment:
I think it might help to add a "Device" template variable, to allow users to scope to a specific device.

Reply:
True! Do you think it should also be used to filter Netflow metrics (by Exporter IP) or should we also add a Exporter template variable to avoid confusion?